src/components/SiderMenu/index.tsx

```diff
@@ -17,13 +17,13 @@ function SiderMenuWrapper (props: SiderMenuWrapperProps) {
     <Drawer
       visible={!collapsed}
       placement='left'
-      onClose={() => onCollapse(true)}
+      onClose={() => onCollapse(true)}  // tslint:disable-line
```

Comment:
these ts lint issues are resolved in the branch #9 that was merged. Please revert